src/endpoint/s3/ops/s3_post_bucket_delete.js

```diff
@@ -22,6 +23,11 @@ async function post_bucket_delete(req) {
         objects
     });
 
+    if (objects.length > 1000) {
+        dbg.error('cannot delete more than 1000 objects');
+        throw new S3Error(S3Error.MalformedACLError);
```

Comment:
Why do you check the number of objects after getting the reply and not before?
* By 'getting the reply' I mean lines 21-24 in your PR.

Reply:
this is a big mistake. I'll fix it